src/core/uint.js

```diff
@@ -217,16 +216,7 @@ UInt.prototype.parse_hex = function(j) {
 };
 
 UInt.prototype.parse_bits = function(j) {
-  if (sjcl.bitArray.bitLength(j) === this.constructor.width * 8) {
-    const bytes = sjcl.codec.bytes.fromBits(j);
-    this.parse_bytes(bytes);
-  } else {
-    this._value = NaN;
-  }
-
-  this._update();
-
-  return this;
+  return this.parse_bytes(sjclcodec.bytes.fromBits(j));
```

Comment:
I think we should fully eliminate sjcl. I don't know what the UTF-8 codec does exactly, but it sounds like it's probably only a few lines of code.